app/src/utils/md.ts

```diff
@@ -1,15 +1,26 @@
 import { marked } from 'marked';
 import dompurify from 'dompurify';
 
+const renderer = new marked.Renderer();
+renderer.link = function (href, title, text) {
+	const link = marked.Renderer.prototype.link.apply(this, [href, title, text]);
+	return link.replace('<a', "<a target='_blank'");
+};
+
 /**
  * Render and sanitize a markdown string
  */
-export function md(str: string): string {
+export function md(str: string, newTab = false): string {
```

Comment:
Lets use an options object rather than a single boolean flag for the "newTab" (and lets rename it to something more descriptive:

```js
// before:
md('# Hello!', true);

// after:

md('# Hello!', { target: "_blank" });